src/app/shared/components/managers-page/managers-page.component.html

```diff
@@ -0,0 +1,52 @@
+<div>
+  <h1 class="page-subtitle">
+    {{'SHARED.COMPONENTS.MANAGERS_PAGE.TITLE' | translate}}
+  </h1>
+  <app-refresh-button (refresh)="changeUser()"></app-refresh-button>
+  <button mat-flat-button color="accent" class="mr-2" (click)="selected === 'user' ? addManager() : addGroup()">
+    {{'SHARED.COMPONENTS.MANAGERS_PAGE.ADD' | translate}}
+  </button>
+  <button mat-flat-button color="warn" (click)="selected === 'user' ? removeManager() : removeGroup()"
+          [disabled]="selectionUsers.selected.length === 0 && selectionGroups.selected.length === 0">
+    {{'SHARED.COMPONENTS.MANAGERS_PAGE.REMOVE' | translate}}
+  </button>
+
+  <mat-form-field class="ml-2">
+    <mat-label>{{'SHARED.COMPONENTS.MANAGERS_PAGE.SELECT_MODE' | translate}}</mat-label>
+    <mat-select [(value)]="selected" (selectionChange)="changeUser()">
+      <mat-option value="user">{{'SHARED.COMPONENTS.MANAGERS_PAGE.USER' | translate}}</mat-option>
+      <mat-option value="group">{{'SHARED.COMPONENTS.MANAGERS_PAGE.GROUP' | translate}} </mat-option>
+    </mat-select>
+  </mat-form-field>
+
+  <mat-form-field class="ml-2">
+    <mat-label>{{'SHARED.COMPONENTS.MANAGERS_PAGE.SELECT_ROLE' | translate}}</mat-label>
+    <mat-select [(value)]="selectedRole" (selectionChange)="changeUser()">
+      <mat-option *ngFor="let role of availableRoles" [value]="role.value">{{role.viewValue | translate}}</mat-option>
```

Comment:
This solution using the DisplayedRole works well but if you would like to make it better, you can use a custom pipe. 
Then it would look something like this: 
```suggestion
      <mat-option *ngFor="let role of availableRoles" [value]="role.value">{{role | displayedRole}}</mat-option>
```
In the displayed role pipe you can inject the translation service and you can define all display values in one place.